cli/src/program.rs

```diff
@@ -236,7 +239,16 @@ impl ProgramSubCommands for App<'_, '_> {
                                     "Use the designated program id even if the account already \
                                      holds a large balance of SOL",
                                 ),
-                        ),
+                        )
+                        .arg(
+                            Arg::with_name("compute_unit_price")
```

Comment:
prefer the pre-defined `Arg` from `clap-utils`

https://github.com/anza-xyz/agave/blob/a2579d484efd78099ba67db5911b2ba12c77b711/clap-utils/src/compute_unit_price.rs#L9-L15

Reply:
resolved